tfhe/benches/shortint/bench.rs

```diff
@@ -638,33 +640,33 @@ define_server_key_scalar_div_bench_fn!(
 );
 
 criterion_group!(
-    arithmetic_operation,
-    unchecked_neg,
-    unchecked_add,
-    unchecked_sub,
-    unchecked_mul_lsb,
-    unchecked_mul_msb,
-    unchecked_div,
+    smart_ops,
     smart_bitand,
     smart_bitor,
     smart_bitxor,
     smart_add,
     smart_sub,
     smart_mul_lsb,
+    programmable_bootstrapping
```

Comment:
no need to add it here it's already in the unchecked ops and it's a kind of unchecked op so fine like this !